modules/covid_vaccine/config/initializers/statsd.rb

```diff
@@ -6,3 +6,18 @@
   StatsD.increment("api.vetext.#{endpoint}.total", 0)
   StatsD.increment("api.vetext.#{endpoint}.fail", 0)
 end
+
+CovidVaccine::V0::RegistrationService.extend StatsD::Instrument
+CovidVaccine::V0::RegistrationService.statsd_measure :attributes_from_mpi,
+                                                     'covid_vaccine.mpi_query.measure'
+CovidVaccine::V0::RegistrationService.statsd_measure :submit,
+                                                     'covid_vaccine.vetext_submit.measure'
+CovidVaccine::V0::RegistrationService.statsd_count_success :attributes_from_mpi,
+                                                           'covid_vaccine.mpi_query', &:present?
+CovidVaccine::V0::RegistrationService.statsd_count_success :submit,
+                                                           'covid_vaccine.vetext_submit'
+
+StatsD.increment('covid_vaccine.mpi_query.success', 0)
+StatsD.increment('covid_vaccine.mpi_query.failure', 0)
```

Comment:
I've seen in other places that we've been defining STATSD_KEY_PREFIX on some of the service classes for statsd keys. In this case, you could create a constant for covid_vaccine.mpi_query and covid_vaccine.vetext_submit

Reply:
Since it's all restricted to one place in this statsd file right now, I did not see much value in defining a constant just to use it twice immediately below, but if this winds up spanning out into other files much further I'll definitely refactor that.